components/raftstore-v2/src/worker/tablet_gc.rs

```diff
@@ -156,10 +156,15 @@ impl<EK: KvEngine> Runner<EK> {
                 "path" => path.display(),
             ),
             Ok(false) => {
+                let (_, region_id, tablet_index) =
+                    registry.parse_tablet_name(path).unwrap_or(("", 0, 0));
                 // TODO: use a meaningful table context.
```

Comment:
remove?

Reply:
It's not meaningful yet. For example, key range is missing.